src/Sfa.Tl.Matching.Web/Frontend/src/javascripts/employer-search.js

```diff
@@ -1,50 +1,18 @@
 "use strict";
-
+var Searchresult = null;
 var employer = (function () {
-    var businessNameMinLength = 2;
+    var queryMinLength = 2;
 
     accessibleAutocomplete.enhanceSelectElement({
         defaultValue: "",
         autoSelect: true,
         selectElement: document.querySelector("#CompanyName"),
-        minLength: businessNameMinLength,
+        minLength: queryMinLength,
         source: search,
         name: "CompanyName",
         onConfirm: setSelectedEmployerId
     });
 
-    $("#tl-continue").click(function () {
```

Comment:
We need this as it persists the value when clicking Continue. There was a ticket raised for it.

Reply:
Finally found the ticket - TLWP-504